extensions/mssql/src/mssql.d.ts

```diff
@@ -1179,6 +1179,12 @@ declare module 'mssql' {
 		databaseFilePaths: string[];
 		owner: string;
 	}
+
+	export interface RestoreParams extends azdata.RestoreInfo {
```

Comment:
Remove and just use azdata ones directly

Reply:
missed removing here!